.buildkite/pipeline.yml

```diff
@@ -7,40 +7,46 @@ env:
   cur_date: '$(date --date="$(git show --no-patch --format=%ci)" +"%Y%m%d%H%M")'
 
 steps:
- - command: nix run nixpkgs.reuse -c reuse lint
-   label: reuse lint
- - command: .buildkite/check-trailing-whitespace.sh
-   label: check trailing whitespace
- - command: "nix run -f https://github.com/serokell/crossref-verifier/archive/68a1f9d25b6e7835fea8299b18a3e6c61dbb2a5c.tar.gz -c crossref-verify"
-   label: crossref-verify
+ - label: reuse lint
+   command: nix run nixpkgs.reuse -c reuse lint
+ - label: check trailing whitespace
+   command: .buildkite/check-trailing-whitespace.sh
+ - label: crossref-verify
+   command: "nix run -f https://github.com/serokell/crossref-verifier/archive/68a1f9d25b6e7835fea8299b18a3e6c61dbb2a5c.tar.gz -c crossref-verify"
    soft_fail: true
- - commands:
+ - label: build and package via nix
```

Comment:
Can you clarify what's going on in this step? AFAIU our dep and rpm packages contain docker-built binaries, don't they? If they do, shouldn't we build with docker before this step?

Reply:
no, they don't, we don't have any docker-based packaging right now

Reply:
Ah, I thought that everything that goes into release is built without nix. And it's actually true, I just forgot that we stopped putting deb and rpm packages into releases.